addons/repair/functions/fnc_isInRepairFacility.sqf

```diff
@@ -31,7 +31,7 @@ private _checkObject = {
     };
 };
 
-private _objects = (lineIntersectsWith [_object modelToWorldVisual [0, 0, (_position select 2)], _object modelToWorldVisual [0, 0, (_position select 2) +10], _object]);
+private _objects = (lineIntersectsSurfaces [_object modelToWorldVisual [0, 0, (_position select 2)], _object modelToWorldVisual [0, 0, (_position select 2) +10], _object]) apply {_x select 3};
```

Comment:
lineIntersectsWith would return multiple entries
lineIntersectsSurfaces  param maxResults defaults to only 1 object

don't see any reason to change this as we don't need terrain or intersection positions

Reply:
Reverted